src/components/filters/FiltersItem.js

```diff
@@ -80,13 +92,20 @@ class FiltersItem extends Component {
   };
 
   mergeData = (property, value, valueTo) => {
+    const DATE_FIELD = "DateDoc";
```

Comment:
i think we shall not use hardcoded field names. DateDoc it's just one of the many Date field names.